app/controllers/sitemaps_controller.rb

```diff
@@ -16,9 +16,6 @@ def sitemap_xml
     listings = Force::ListingService.listings
     @sitemap = SitemapGenerator::Builder::SitemapFile.new(host: 'https://' + host_name)
     @sitemap.add '/', changefreq: 'weekly', priority: 1.0
-    @sitemap.add '/welcome-chinese', changefreq: 'weekly'
```

Comment:
we removed these pages as part of the home page rewrite, but hadn't removed them from this site map